auction_seller_invoice/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2022 Quartile Limited
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Auction Seller Invoice",
+    "category": "Auction",
+    "version": "15.0.1.0.0",
+    "author": "Quartile Limited",
+    "website": "https://www.quartile.co",
+    "license": "AGPL-3",
+    "depends": [
+        "auction_seller",
+        "account",
+        "account_invoice_extract",
```

Comment:
@AungKoKoLin1997 What's this dependency about?

Reply:
This dependency is for adding access right of models to auction seller group. And I am also thinking about license. "account_invoice_extract" module is the OEEL license and "account_commission" is the AGPL license. And I need to depend "account_invoice_extract" and add access right of "account_invoice_extract" model to auction seller group. I am not sure this module license should be which license.

Reply:
The client uses CE, so please just remove this dependency.